hieradata/dev/wso2/wso2am/1.10.0/mesos/default.yaml

```diff
@@ -34,7 +34,8 @@ wso2::mgt_hostname: marathon-lb.marathon.mesos
 wso2::clustering:
   enabled: true
   membership_scheme: mesos
-#  mesos:
+  mesos:
+    marathon_applications: wso2am-gateway-manager,wso2am-gateway-worker
```

Comment:
Both manager and worker has the same marathon applications since they refer to each other for clustering. Is n't it?
"marathon_applications: wso2am-gateway-manager,wso2am-gateway-worker"